### PR TITLE
Verify RGW VIP is properly configured

### PR DESCRIPTION
**Pull request by fmount**

### CODE REVIEW

**Comment by fultonj** [comment]
```
2024-09-30 18:00:02.630 20 CRITICAL tempest [-] Unhandled error: config_tempest.services.base.ServiceError: Request on service 'share' with url 'https://manila-public-openstack.apps-crc.testing/v1/5bdeea74f87e49f5a3695883e78be47d/scheduler-stats/pools/detail' failed with code 401
2024-09-30 18:00:02.630 20 ERROR tempest Traceback (most recent call last):
2024-09-30 18:00:02.630 20 ERROR tempest   File "/usr/bin/discover-tempest-config", line 10, in <module>
2024-09-30 18:00:02.630 20 ERROR tempest     sys.exit(main())
2024-09-30 18:00:02.630 20 ERROR tempest   File "/usr/lib/python3.9/site-packages/config_tempest/main.py", line 617, in main
2024-09-30 18:00:02.630 20 ERROR tempest     config_tempest(
2024-09-30 18:00:02.630 20 ERROR tempest   File "/usr/lib/python3.9/site-packages/config_tempest/main.py", line 559, in config_tempest
2024-09-30 18:00:02.630 20 ERROR tempest     services = Services(clients, conf, credentials)
2024-09-30 18:00:02.630 20 ERROR tempest   File "/usr/lib/python3.9/site-packages/config_tempest/services/services.py", line 42, in __init__
2024-09-30 18:00:02.630 20 ERROR tempest     self.discover()
2024-09-30 18:00:02.630 20 ERROR tempest   File "/usr/lib/python3.9/site-packages/config_tempest/services/services.py", line 122, in discover
2024-09-30 18:00:02.630 20 ERROR tempest     service.set_default_tempest_options(self._conf)
2024-09-30 18:00:02.630 20 ERROR tempest   File "/usr/lib/python3.9/site-packages/config_tempest/services/share.py", line 48, in set_default_tempest_options
2024-09-30 18:00:02.630 20 ERROR tempest     pools = self.get_share_pools(detail=True)['pools']
2024-09-30 18:00:02.630 20 ERROR tempest   File "/usr/lib/python3.9/site-packages/config_tempest/services/share.py", line 37, in get_share_pools
2024-09-30 18:00:02.630 20 ERROR tempest     body = self.do_get(url)
2024-09-30 18:00:02.630 20 ERROR tempest   File "/usr/lib/python3.9/site-packages/config_tempest/services/base.py", line 80, in do_get
2024-09-30 18:00:02.630 20 ERROR tempest     raise ServiceError("Request on service '%s' with url '%s' failed"
2024-09-30 18:00:02.630 20 ERROR tempest config_tempest.services.base.ServiceError: Request on service 'share' with url 'https://manila-public-openstack.apps-crc.testing/v1/5bdeea74f87e49f5a3695883e78be47d/scheduler-stats/pools/detail' failed with code 401
2024-09-30 18:00:02.630 20 ERROR tempest [00m
```

https://logserver.rdoproject.org/08/2408/8a1d8b2df05c0c4ec97d7736497dcaf04e0775e4/github-check/podified-multinode-hci-deployment-crc/2950e0b/controller/ci-framework-data/logs/openstack-k8s-operators-openstack-must-gather/namespaces/openstack/pods/tempest-tests-bm4mn/logs/tempest-tests-tests-runner.log

**Comment by fmount** [comment]
Endpoints look good, and tempest worked [1]

[1] https://logserver.rdoproject.org/08/2408/e84c11b87bc451a4b35e790d6efa9507d3f4775d/github-check/podified-multinode-hci-deployment-crc/4b1dfa4/controller/ci-framework-data/logs/openstack-k8s-operators-openstack-must-gather/ctlplane/endpoints

File: roles/cifmw_cephadm/tasks/post.yml [hunk 1]
```diff
@@ -46,8 +46,14 @@
       loop: "{{ cifmw_cephadm_log_commands }}"
 
 - name: Configure ceph object store to use external ceph object gateway
+  when:
+    - cifmw_cephadm_vip is defined
+    - cifmw_cephadm_vip default("") | length > 0
```
Comment (fultonj) [inline roles/cifmw_cephadm/tasks/post.yml:51]: `cifmw_cephadm_vip | default("") | length > 0`

Comment (fmount) [reply roles/cifmw_cephadm/tasks/post.yml]: Thank you! Fixed

**Comment by fultonj** [comment]
Want to rebase on https://github.com/openstack-k8s-operators/architecture/pull/410 ?

**Comment by fmount** [comment]
rebased

**Review by fultonj — COMMENTED** [review]
I'm +2 to merge this. 

Let's just make sure the github-experimental check passes first.

**Comment by pablintino** [comment]
The testproject passed
/approve

**Comment by fultonj** [comment]
Should we cherry pick this to the next release?

**Comment by fmount** [comment]
> Should we cherry pick this to the next release?

Makes sense to me.

**Pull request merged**